tools/notifyServicelogEvent.c

```diff
@@ -140,77 +143,56 @@ main(int argc, char *argv[])
 
     if (sscanf(argv[optind], "%" U64TS, &event_id) != 1) {
         crm_err("Error: could not read event_id from args!");
-
         rc = 1;
         goto done;
     }
 
     if (event_id == 0) {
         crm_err("Error: event_id is 0!");
-
         rc = 1;
         goto done;
     }
 
     rc = servicelog_open(&slog, 0);     /* flags is one of SL_FLAG_xxx */
-
-    if (!slog) {
-        crm_err("Error: servicelog_open failed, rc = %d", rc);
-
+    if (rc != 0) {
```

Comment:
I could revert this, I don't have strong feelings. If `slog == NULL`, then rc is nonzero, and vice-versa.
https://github.com/power-ras/libservicelog/blob/master/src/servicelog.c#L555-L588

Reply:
I'm fine with relying on rc, but if you want to be paranoid it's easy to do (slog == NULL) || (rc != 0)